### DISCUSSION TITLE
Brakelight support

### DISCUSSION THREAD

**Discussion by aalm** [danielnilsson9/bbs-fw#147] [Ideas]
I intend to use the light output for controlling a brakelight(with a 6V relay).
To me the light output seems relatively underpowered, so I want the actual lights to run on a different battery anyway.
And this seems to be the most reasonable use for the light output(beyond using it for debugging).

Thought I'd ask if there's any interest before making a pull request?

Here's an commit, which it might look like on top of master-branch, fwiw.:
https://github.com/aalm/bbs-fw/commit/7b396081a5b28a9b2738543e0c16ffe6a1f50439

**Comment by dav0000000** [comment]
Sounds like a innovative idea. I run a headlight and taillight both from the 6v. I agree the headlight is pretty useless but it is adequate to meet the "always on" ebike rule in my country and is also ok riding at night with streetlights. I have another 15w high beam/low beam headlight that is perfect for trails which I ride nearly every night, it runs directly from the 52v battery with a switch on the handlebars. The 6v taillight is bright enough though. The taillight has a built in brake light which I haven't wired to anything, it would be cool to have it work but for me, not at the expense of losing the taillight and headlight turning on automatically. And 3 watts for tail and headlight is a big win. Can a brake light not be powered from the brake circuit directly or is the power back to front meaning the circuit has power when there is no braking and is cut when braking. I guess that would require something special to make it work. That would be my preference.

**Comment by aalm** [comment]
My brakelight test-setup working as it should: https://www.youtube.com/shorts/WpjVhp5wI4k
Display is pretty slow to show the icon for braking. I used "usb c pd to 12v"-cable with powerbank.

For the "always on"-configuration, I'd rather get lights that can be plugged directly to the battery
somehow, and have those be on whenever battery is "on". That way I'd never forget to leave it on.

**Reply by dav0000000** [reply]
All good. My battery is always turned on even though it has an on/off switch. I still really like the idea of a brake light though.
